app/assets/stylesheets/admin.scss

```diff
@@ -128,6 +128,17 @@ label {
     height: 40px;
     width: 40px;
   }
+
+  &--label {
+    .enabled-indicator-hidden {
+      display: none;
+    }
+
+    .enabled-indicator-visible {
+      display: flex;
+      align-items: center;
+    }
+  }
```

Comment:
any reason not to make these classes just `hidden` and `visible`? 

**also, this is not a blocker, but...** instead of making `display: none;` a class, you could just make that the _default_, and then only toggle the visible class, which would allow you to leverage CSS here _even further_! 

```css
&--label {
  display: none;
  
  .visible {
    display: flex;
    align-items: center;
  }
}
```

I'm not 100% sure that _exact_ CSS is correct (I haven't tried it locally), but something like that _should_ work.

Then, your JS would just only ever have to do this:
```javascript
this.enabledIndicatorTarget.classList.toggle('visible');
```

since `display: none` would be the _default_ you wouldn't have to go through the effort of adding `enabled-indicator-hidden` to the `<div>` like you do [here](https://github.com/forem/forem/pull/11185/commits/f4763660d3f1e9845deecad3e2662849425118ad#diff-892ec00ceea1d889fc3c52af2033fa1932e7c61d267d7acfc1027ae5b5029e0cR177) and [here](https://github.com/forem/forem/pull/11185/commits/f4763660d3f1e9845deecad3e2662849425118ad#diff-892ec00ceea1d889fc3c52af2033fa1932e7c61d267d7acfc1027ae5b5029e0cR281).

Instead, it would look like this:
```html
<div class="<%= SiteConfig.allow_both_email_signup_and_login ? "visible" : "" %>"
```
The beauty of this is that you're leaning on CSS's "cascade" (_cascading style sheets_) and defining the _default_ style _within the `&--label` class itself_, rather than having to define a `hidden` class and then applying it as the default :)

I'm happy to pair on this with you in a separate PR if you're curious and would like to figure it out together!